cyclopts/help/converters.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from rich.console import RenderableType
+
+    from cyclopts.help import AbstractTableEntry
+
+
+def asterisk_converter(_: RenderableType, inp: AbstractTableEntry) -> RenderableType:
+    if inp.required:
+        return "*"
+    return ""
+
+
+def stretch_name_converter(_: RenderableType, inp: AbstractTableEntry) -> RenderableType:
```

Comment:
I'm a big worried by the existence of this. I haven't gotten to how it's used yet, but it would suggest that names are being combined into a single string somewhere else.

Reply:
got it, i think your current implementation is actually totally reasonable.